### PR TITLE
SITL: document GPS parameters

### PR DESCRIPTION
**Pull request by antholuo**

### CODE REVIEW

File: libraries/SITL/SITL.cpp [hunk 1]
```diff
@@ -472,28 +477,82 @@ const AP_Param::GroupInfo SIM::var_gps[] = {
...
+    // @Units: %
     // @User: Advanced
     AP_GROUPINFO("GPS_BYTELOSS",   4, SIM,  gps_byteloss[0],  0),
     // @Param: GPS_NUMSATS
     // @DisplayName: GPS 1 Num Satellites
     // @Description: Number of satellites GPS 1 has in view
     AP_GROUPINFO("GPS_NUMSATS",    5, SIM,  gps_numsats[0],   10),
+    // @Param: GPS_GLITCH
+    // @DisplayName: GPS 1 Glitch
+    // @Description: Glitch offsets of simulated GPS 2 sensor
```
Comment (rmackay9) [inline libraries/SITL/SITL.cpp:489]: Txs for this.  I guess it's "GPS 1 sensor"?

Comment (antholuo) [reply libraries/SITL/SITL.cpp]: oops - I messed that one up. Should be fixed now!

File: libraries/SITL/SITL.cpp [hunk 2]
```diff
@@ -503,27 +562,88 @@ const AP_Param::GroupInfo SIM::var_gps[] = {
...
+    // @Values: 0:Disabled, 1:Enabled
+    // @User: Advanced
     AP_GROUPINFO("GPS2_HDG",      42, SIM,  gps_hdg_enabled[1], SIM::GPS_HEADING_NONE),
+    // @Param: GPS2_ACC
+    // @DisplayName: GPS 2 Accuracy
+    // @Description: GPS 2 Accuracy
+    // @Vector3Parameter: 1
+    // @User: Advanced
     AP_GROUPINFO("GPS2_ACC",      43, SIM,  gps_accuracy[1], 0.3),
+    // @Param: GPS_VERR
```
Comment (rmackay9) [inline libraries/SITL/SITL.cpp:624]: Ah, this should be GPS2_VERR

**Comment by peterbarker** [comment]
@antholuo you can run the relevant autotest yourself like this:
```
./Tools/autotest/atutotest.py build.Rover
./Tools/autotest/atutotest.py test.Rover.LoggerDocumentation
```

You only need to run the second line if the first one passes and the documentation test fails.

File: libraries/SITL/SITL.cpp (hunk 2 above)
Comment (antholuo) [reply libraries/SITL/SITL.cpp]: Ahh another mistake from me, 

This has also been fixed.

**Review by rmackay9 — APPROVED** [review]
LGTM, thanks!

File: libraries/SITL/SITL.cpp [hunk 3]
```diff
@@ -472,28 +477,82 @@ const AP_Param::GroupInfo SIM::var_gps[] = {
...
+    // @Param: GPS_HDG
+    // @DisplayName: GPS 1 Heading
+    // @Description: Enable GPS1 output of NMEA heading HDT sentence or UBLOX_RELPOSNED
+    // @Values: 0:Disabled, 1:Enabled
+    // @User: Advanced
     AP_GROUPINFO("GPS_HDG",       13, SIM,  gps_hdg_enabled[0], SIM::GPS_HEADING_NONE),
+    // @Param: GPS_ACC
+    // @DisplayName: GPS 1 Accuracy
+    // @Description: GPS 1 Accuracy
+    // @Vector3Parameter: 1
```
Comment (peterbarker) [inline libraries/SITL/SITL.cpp:536]:
```suggestion
```

not a vector3.  same for acc2

**Review by peterbarker — CHANGES_REQUESTED** [review]
The remaining INIT_ALT_* things has me puzzled.  Looking into it

File: libraries/SITL/SITL.cpp [hunk 4]
```diff
@@ -503,27 +562,88 @@ const AP_Param::GroupInfo SIM::var_gps[] = {
...
+    // @User: Advanced
     AP_GROUPINFO("GPS2_VERR",     44, SIM,  gps_vel_err[1], 0),
 
+    // @Param: INIT_LAT_OFS
+    // @DisplayName: Initial Latitude Offset
+    // @Description GPS initial lat offset from origin
     AP_GROUPINFO("INIT_LAT_OFS",  45, SIM,  gps_init_lat_ofs, 0),
+    // @Param: INIT_LON_OFS
+    // @DisplayName: Initial Longitude Offset
+    // @Description GPS initial lon offset from origin
```
Comment (peterbarker) [inline libraries/SITL/SITL.cpp:637]:
```suggestion
    // @Description: GPS initial lon offset from origin
```

File: libraries/SITL/SITL.cpp [hunk 5]
```diff
@@ -503,27 +562,88 @@ const AP_Param::GroupInfo SIM::var_gps[] = {
...
+    // @Param: GPS2_VERR
+    // @DisplayName: GPS 2 Velocity Error
+    // @Description: GPS 2 Velocity Error Offsets in NED
+    // @Vector3Parameter: 1
+    // @User: Advanced
     AP_GROUPINFO("GPS2_VERR",     44, SIM,  gps_vel_err[1], 0),
 
+    // @Param: INIT_LAT_OFS
+    // @DisplayName: Initial Latitude Offset
+    // @Description GPS initial lat offset from origin
```
Comment (peterbarker) [inline libraries/SITL/SITL.cpp:633]:
```suggestion
    // @Description: GPS initial lat offset from origin
```

**Review by peterbarker — CHANGES_REQUESTED** [review]
Found the other problem

**Review by peterbarker — APPROVED** [review]

**Comment by peterbarker** [comment]
Merged, thanks!

**Pull request merged**